books/rtl/rel11/support/reps.lisp

```diff
@@ -819,7 +819,7 @@
   :prep-lemmas (
     (defrule lemma
       (implies (and (posp p)
-                    (real/rationalp x)
+                    (rationalp x)
```

Comment:
Lemma with real/rationalp passes in ACL2 byut fails in ACL2(r).
Lemma rationalp passes in both of them.